src/vs/workbench/contrib/tasks/common/jsonSchema_v2.ts

```diff
@@ -378,6 +388,8 @@ const taskConfiguration: IJSONSchema = {
 			default: false
 		},
 		presentation: Objects.deepClone(presentation),
+		color: Objects.deepClone(color),
```

Comment:
Instead of putting these at the top level of a task, I would consider having a `icon` property which has an `id` and `color`, as I suspect that if this is made tasks API that's what we'd go with.

Reply:
It's how all theme icons work in the vscode.d.ts.